HowToProfileLedger.md

```diff
@@ -1,67 +1,68 @@
-                        How To Profile Ledger
-                            last edited
-                            May 7, 2022
-                            Tim Sheard
+# How To Profile Ledger
+*Tim Sheard*
```

Comment:
Probably left by accident? :D  We can surely see the authors from git history

Reply:
Yeah, but I figured since Tim wrote it, I'll leave it in. Not sure how to format it nicely though.